projects/pgai/tests/semantic_catalog/test_manage.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+
+import psycopg
+import pytest
+
+import pgai
+from pgai.semantic_catalog import semantic_catalog
+from tests.semantic_catalog.utils import PostgresContainer
+
+DATABASE = "sc_04"
```

Comment:
Looks like we now have 3 test modules that use the same database `sc_04`. I wonder if this is a race condition in the tests. I guess we aren't running tests concurrently. Maybe we should switch to having the database named after the module (e.g. `test_manage`). That way, if you run all the tests, the evidence isn't dropped by another test (makes debugging easier).

Reply:
Done in b121060d4bbf5d612687769a490de19ececbf103. `__name__ == "tests.semantic_catalog.test_manage"` so doing the split and grabbing last element to get just the `test_manage` bit.